### PR TITLE
update image classification pretrained models

### PR DESCRIPTION
**Pull request by shippingwang**
- [x] update new pre-trained models,include 

image

- [x] add mixup and label smoothing etc. args

### CODE REVIEW

File: PaddleCV/image_classification/README.md [hunk 1]
```diff
@@ -149,21 +149,31 @@ Available top-1/top-5 validation accuracy on ImageNet 2012 are listed in table.
...
-|[ResNet50](http://paddle-imagenet-models-name.bj.bcebos.com/ResNet50_pretrained.tar) | 76.35%/92.80% | 76.22%/92.92% |
-|[ResNet101](http://paddle-imagenet-models-name.bj.bcebos.com/ResNet101_pretrained.tar) | 77.49%/93.57% | 77.56%/93.64% |
-|[ResNet152](https://paddle-imagenet-models-name.bj.bcebos.com/ResNet152_pretrained.tar) | 78.12%/93.93% | 77.92%/93.87% |
-|[SE_ResNeXt50_32x4d](https://paddle-imagenet-models-name.bj.bcebos.com/SE_ResNeXt50_32x4d_pretrained.tar) | 78.50%/94.01% | 78.44%/93.96% |
-|[SE_ResNeXt101_32x4d](https://paddle-imagenet-models-name.bj.bcebos.com/SE_ResNeXt101_32x4d_pretrained.tar) | 79.26%/94.22% | 79.12%/94.20% |
-|[GoogleNet](https://paddle-imagenet-models-name.bj.bcebos.com/GoogleNet_pretrained.tar) | 70.50%/89.59% | 70.27%/89.58% |
-|[ShuffleNetV2](https://paddle-imagenet-models-name.bj.bcebos.com/ShuffleNetV2_pretrained.tar) |  | 69.48%/88.99% |
+|model | top-1/top-5 accuracy(CV2) |
+|- |:-: |
+|[AlexNet](http://paddle-imagenet-models-name.bj.bcebos.com/AlexNet_pretrained.tar) | 56.72%/79.17% |
```
Comment (qingqing01) [inline PaddleCV/image_classification/README.md:154]: 这里标题不太合理吧，56.71是PIL测出来的结果，并且CV2

File: PaddleCV/image_classification/reader_cv2.py [hunk 2]
```diff
@@ -50,27 +53,29 @@ def random_crop(img, size, scale=None, ratio=None):
...
 
-
 def resize_short(img, target_size):
     """ resize_short """
     percent = float(target_size) / min(img.shape[0], img.shape[1])
     resized_width = int(round(img.shape[1] * percent))
     resized_height = int(round(img.shape[0] * percent))
-    resized = cv2.resize(img, (resized_width, resized_height), interpolation=cv2.INTER_LANCZOS4)
+    resized = cv2.resize(img, (resized_width, resized_height), 
+            #interpolation=cv2.INTER_LANCZOS4
```
Comment (qingqing01) [inline PaddleCV/image_classification/reader_cv2.py:75]: remove commented code

File: PaddleCV/image_classification/train.py [hunk 3]
```diff
@@ -178,19 +191,69 @@ def net_config(image, label, model, args):
...
+            cost = fluid.layers.cross_entropy(input=softmax_out2, label=smooth_out1, soft_label=True) 
 
-        acc_top1 = fluid.layers.accuracy(input=pred, label=label, k=1)
-        acc_top5 = fluid.layers.accuracy(input=pred, label=label, k=5)
+            avg_cost = fluid.layers.mean(cost)
+            acc_top1 = fluid.layers.accuracy(input=softmax_out2, label=label, k=1)
+            acc_top5 = fluid.layers.accuracy(input=softmax_out2, label=label, k=5)
 
-    return avg_cost, acc_top1, acc_top5
+            return avg_cost, acc_top1, acc_top5
```
Comment (qingqing01) [inline PaddleCV/image_classification/train.py]: 上面分支太多，可以考虑定义不同的子函数，这块调用

File: PaddleCV/image_classification/train.py [hunk 4]
```diff
@@ -201,20 +264,66 @@ def build_program(is_train, main_prog, startup_prog, args):
...
+            #    image = fluid.layers.cast(image, "float16")
+            
+            """
+            if is_train:
+                if use_mixup:
+                    avg_cost = net_config(image=image, y_a=y_a, y_b=y_b, lam=lam, model=model, args=args, label=0, is_train=True)
+                else:
+                    avg_cost, acc_top1, acc_top5 = net_config(image, model, args, label=label, is_train=True)
+            else:
+                avg_cost, acc_top1, acc_top5 = net_config(image, model, args, is_train=false, label)
```
Comment (qingqing01) [inline PaddleCV/image_classification/train.py]: 去掉无用的code

File: PaddleCV/image_classification/README.md (hunk 1 above)
Comment (cuicheng01) [reply PaddleCV/image_classification/README.md:154]: 这个是cv2测的，之前的cv2有bug

File: PaddleCV/image_classification/train.py (hunk 3 above)
Comment (cuicheng01) [reply PaddleCV/image_classification/train.py]: 这块主要原因是使用mixup以后整个逻辑都不太一样了，train和test的也不太一样，后续可以想办法优化一下代码

**Review by qingqing01 — APPROVED** [review]
